spec/bot.intergration.ts

```diff
@@ -64,18 +65,19 @@ const mockAnalyzerClient: AnalyzerClient = {
   }
 };
 
+const repliers = [
+  docsClient,
+  eventsClient,
+]
 const analyzers = [mockAnalyzerClient];
 
-commandTree.registerSubCommand(docsClient.commandSubtree);
-commandTree.registerSubCommand(eventsClient.commandSubtree);
-
 const dummyMessage: MessageModel = {
   text: 'some text here'
 };
 
-describe(`Angie`, () => {
+describe(`Bot`, () => {
 
-  const angie = new Angie(null, null, true, commandTree, analyzers, 0, null, null);
+  const angie = new Bot(null, null, false, repliers, analyzers);
```

Comment:
Just pass in an object with named keys?

```
new Bot({foo: null, bar: null, bool: flase, repliers, analyzers})
```

Look at [how nice it can look like](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#ES2015_version).